src/index.html

```diff
@@ -1,13 +1,406 @@
+              <li class="features__item">Auto Music Playback from Last Song Stopped</li>
+            </ul>
+          </div>
+
+          <div class="features__card features__card--2">
+            <h3 class="features__feature">Connectivity</h3>
+            <ul class="features__feature-list">
+              <li class="features__item">Hands Free Wireless Audio</li>
+              <li class="features__item">Bluetooth 4.0 LE</li>
+              <li class="features__item">Wi-Fi 2.4 GHz (802.11 b/g/n)</li>
+              <li class="features__item">Smart Multiroom System Set Up</li>
+              <li class="features__item">Party Mode with 6.0 Units and above</li>
+              <li class="features__item">MESHNET Multi Speaker Network</li>
+            </ul>
+          </div>
+
+          <div class="features__card features__card--3">
+            <h3 class="features__feature">App Features</h3>
+            <ul class="features__feature-list">
+              <li class="features__item">Customize Music Schedule</li>
+              <li class="features__item">Wake Up with Favorite Songs</li>
+              <li class="features__item">Home Detection Auto Wake Up</li>
+              <li class="features__item">Color Wheel</li>
+            </ul>
+          </div>
+
+          <div class="features__slider">
+            <div class="features__slider-arrows">
+              <svg
+                class="features__slider-left"
+                width="48"
+                height="13"
+                viewBox="0 0 48 13"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path d="M48 4.19628e-06L0.329261 0L0.329261 0.999998L48 1V4.19628e-06Z" fill="#BDBDBD"></path>
+                <path d="M18.2022 11.8047L0.440509 0.000149576L0 1.19548L17.7616 13L18.2022 11.8047Z" fill="#BDBDBD"></path>
+              </svg>
+              <svg
+                class="features__slider-right"
+                width="48"
+                height="13"
+                viewBox="0 0 48 13"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path d="M0 13H47.6707V12H0V13Z" fill="#333333"></path>
+                <path d="M29.7978 1.19533L47.5595 12.9999L48 11.8045L30.2384 0L29.7978 1.19533Z" fill="#333333"></path>
+              </svg>
+            </div>
+            <div class="features__slider-numbers">
+              <span class="features__slider-number-now">01</span> /
+              <span class="features__slider-number-remaining">03</span>
+            </div>
+          </div>
+        </div>
+      </article>
+    </section>
+    <!-- #endregion -->
+    <!-- #region questions -->
+    <section class="questions" id="get-in-touch">
+      <article class="questions__container">
+        <h2 class="questions__title">DO YOU HAVE ANY QUESTIONS?</h2>
+        <form onsubmit="this.reset(); return false;" class="questions__form">
```

Comment:
Form isn't moving to the top of the page. Check the item 10 of checklist.

When you try to send the form there is no 405 error and the form is automatically cleared after submit and is scrolled to the top of the page or the page is reloaded